crates/telio-firewall/src/firewall.rs

```diff
@@ -208,6 +218,26 @@ pub trait Firewall {
         sink4: &mut dyn io::Write,
         sink6: &mut dyn io::Write,
     ) -> io::Result<()>;
+
+    /// Saves local node Ip address into firewall object
+    fn set_ip_address(&self, ip_addrs: Option<Vec<StdIpAddr>>);
+}
+
+/// This enum has to be documented
+#[derive(Clone, Debug, PartialEq)]
+pub enum Permissions {
+    /// this too
+    IncomingConnections,
+    /// and this as well
```

Comment:
and this as well? 😆

Reply:
Had left to complete at the end. Forgot to clean that up 🙈